src/Components/Labels/CategoryLabel.js

```diff
@@ -27,13 +27,14 @@ const CategoryLabel = ({ tags }) => {
   const intl = useIntl();
 
   return (
-    <LabelGroup numLabels={4}>
+    <LabelGroup numLabels={1} isCompact>
       {extractCategories(tags).map((tag, key) => (
         <Label
           key={key}
           icon={CATEGORY_ICONS[tag]}
           variant="outline"
           color="blue"
+          isCompact
```

Comment:
Looks unnecessary. If you define `isCompact` for the `LabelGroup`, there is no need to add the same parameter to each of the nested labels: https://www.patternfly.org/v4/components/label-group/.

Reply:
okk, checked it on my instance and I have the same issue as yours. 👍🏼  It seems like a bug or smth since their documentation states that the labels must become compact once the parameter is provided for the `LabelGroup`... created https://github.com/patternfly/patternfly-react/issues/6815